src/transform/join-commands.ts

```diff
@@ -0,0 +1,3 @@
+export default (commands: string[]): string => {
```

Comment:
This is the function that later will attempt to optimize the joining of commands.